sdk/src/transaction.rs

```diff
@@ -2,12 +2,17 @@
 
 use crate::hash::{Hash, Hasher};
 use crate::pubkey::Pubkey;
+use crate::shortvec::{
+    deserialize_vec, deserialize_vec_with, encode_len, serialize_vec, serialize_vec_with,
+};
 use crate::signature::{Keypair, KeypairUtil, Signature};
-use bincode::serialize;
-use serde::Serialize;
+use bincode::{deserialize_from, serialize, serialize_into, Error};
+use serde::{Deserialize, Serialize, Serializer};
+use std::fmt;
+use std::io::Cursor;
 use std::mem::size_of;
 
-pub const SIG_OFFSET: usize = size_of::<u64>();
+const MAX_SERIALIZED_TX_SIZE: usize = 512;
```

Comment:
sdk/src/packet.rs:pub const PACKET_DATA_SIZE: usize = 512;

Reply:
Updated to use PACKET_DATA_SIZE